components/plurals/src/operands.rs

```diff
@@ -151,11 +162,19 @@ impl FromStr for PluralOperands {
     type Err = OperandsError;
 
     fn from_str(input: &str) -> Result<Self, Self::Err> {
```

Comment:
Discussion: I've wanted for a while to get rid of `impl FromStr for PluralOperands` in favor of `FixedDecimal` which has the same functionality. Since this re-write is not super pretty (you need to convert the bytes back to a string to parse them to an int), maybe just leave this as-is and don't expose it directly over FFI until someone asks?